lib/components/Common/Dropdown/index.js

```diff
@@ -94,8 +94,8 @@ const Dropdown = forwardRef(
     return (
       <div
         ref={wrapperRef}
-        className={classnames("neeto-ui-dropdown__wrapper", {
-          "neeto-ui-dropdown__wrapper--auto-width": autoWidth,
+        className={classnames("nui-dropdown__wrapper", {
```

Comment:
Thoughts about using prefix `ne` instead of `nui`?

Reply:
Sure